airbyte-integrations/connectors/destination-aws-datalake/destination_aws_datalake/config_reader.py

```diff
@@ -5,10 +5,82 @@
 import enum
 
 
-class AuthMode(enum.Enum):
+class CredentialsType(enum.Enum):
     IAM_ROLE = "IAM Role"
     IAM_USER = "IAM User"
 
+    @staticmethod
+    def from_string(s: str):
+        if s == "IAM Role":
+            return CredentialsType.IAM_ROLE
+        elif s == "IAM User":
+            return CredentialsType.IAM_USER
+        else:
+            raise ValueError(f"Unknown auth mode: {s}")
+
+
+class OutputFormat(enum.Enum):
+    PARQUET = "Parquet"
+    JSONL = "JSONL"
+
+    @staticmethod
+    def from_string(s: str):
+        if s == "Parquet":
+            return OutputFormat.PARQUET
+        elif s == "JSONL":
+            return OutputFormat.JSONL
+        else:
+            raise ValueError(f"Unknown output format: {s}")
```

Comment:
for backwards compatibility, let's fall back to one of the values in case `s` is empty.